### ISSUE TITLE
mob_recon 3.0.3+galaxy0 fails with unknown parameter

### ISSUE THREAD

**Issue by hexylena** [phac-nml/galaxy_tools#265]
```
mob_recon: error: unrecognized arguments: --run_typer
```

https://github.com/phac-nml/galaxy_tools/blob/master/tools/mob_suite/mob_recon.xml#L17 this looks old, maybe it was leftover from a mob recon/typer split?

**Comment by kbessonov1984** [comment]
Yes `mob_recon` by default runs `mob_typer` so this update is reasonable. I will be releasing shortly the latest version of the `mob_suite` wrapper as the latest version 3.1.9 has many useful features implemented and is more robust in terms of the results consistency.
